package.json

```diff
@@ -161,5 +161,6 @@
     "turbo": "^1.1.2",
     "typescript": "^4.3.2",
     "xstate-parser-demo": "0.5.3"
-  }
+  },
+  "prettier": {}
```

Comment:
This just helps my IDE to auto-format with Prettier because thanks to that it recognizes that this project is actually using Prettier.